app/views/projects/index.html.slim

```diff
@@ -28,6 +29,10 @@ table.table.table-striped.table-bordered.table-condensed.hidden-sm.hidden-xs
               = icon('minus')
       td = project_tags project
       td
+        - if project.teams.any?
```

Comment:
You can remove the `if` – teams will always return a list. If it's empty, `#each` just won't do anything